app/views/admin/status_edits/_status_edit.html.haml

```diff
@@ -1,20 +1,30 @@
-.status
-  .status__content><
-    - if status_edit.spoiler_text.blank?
-      = prerender_custom_emojis(status_content_format(status_edit), status_edit.emojis)
-    - else
-      %details<
-        %summary><
-          %strong> Content warning: #{prerender_custom_emojis(h(status_edit.spoiler_text), status_edit.emojis)}
-        = prerender_custom_emojis(status_content_format(status_edit), status_edit.emojis)
+%li
+  .history__entry
+    %h5
+      - if status_edit_iteration.first?
```

Comment:
Looks ok, but this kind of implicitly-defined methods is what I dislike the most about Rails. You can kind of guess what it does, but looking up documentation for this is harder than it needs to be. (For the record, this is documented here: https://api.rubyonrails.org/v7.1.3.2/classes/ActionView/PartialRenderer.html#class-ActionView::PartialRenderer-label-Rendering+a+collection+of+partials)